docs/semantics.md

```diff
@@ -377,3 +377,12 @@ Not all of the usual file system features are supported. Most prominently:
 - File and directory permissions and ownership cannot be changed. See the permissions section above.
 - Modification times are not tracked for any inodes except for files.
 - No other times besides modification time are tracked. For example, ctime and atime are not tracked (but will be set to something reasonable). Requests to change them will appear to succeed, but the results are unspecified.
+
+# Incompatibilities with GCS (Google Cloud Storage)
+
+## Unsupported objects
+
+**Scenario** GCS supports creating objects having name or prefix such as `A//B` (in fact object `A//B` can even co-exist with another object `A/B` in the same GCS bucket). Such objects cannot be represented in a gcsfuse-mounted filesystem, because linux filesystem does not support directory named `/`. This causes either an `input/output error` or unexpected behavior on listing the directory entries of `A` (i.e. `ls A`) in a gcsfuse-mounted file-system. Because of the this issue, even objects having name or prefix `A/C` (which are perfectly supported by linux filesystem) cannot be listed by gcsfuse by `ls A`.
```

Comment:
If you can add, you can add as Non Linux Standard FileNames as a heading.

Reply:
that sounds even more confusing to me.

Reply:
Yea I agree. just unsupported objects is fine then. I was trying to add a section for these unsupported ones telling they are Linux dependent. But I understand it may be complicating this.

Reply:
that's why @marcoa6 on this PR to take his opinion on it as this is user-visible documentation.

Reply:
Lets simplify:
Objects in GCS with double slashes '//' as a name or prefix are not supported in GCSfuse. Accessing a directory with such named files will cause an 'input/output error', as the Linux filesystem does not support files or directories named with a '/'. The most common example of this is an object called, for example 'A//C.txt' where 'A' indicates a directory and 'C.txt' indicates a file, and is missing directory 'B/' between 'A/' and 'C.txt'

Reply:
Done. Reduced it to above suggestion from @marcoa6.